configure.ac

```diff
@@ -84,13 +84,19 @@ LIBS="${PTHREAD_LIBS} ${LIBS}"
 CFLAGS="${CFLAGS} ${PTHREAD_CFLAGS}"
 CC="${PTHREAD_CC}"
 
+TS_CHECK_JEMALLOC
+if test "$has_jemalloc" = "0"; then
+	TS_CHECK_TCMALLOC
+fi
+if test "$has_jemalloc$has_tcmalloc" = "00"; then
+	AC_C_MALLOPT
+	AC_C_MALLINFO
```

Comment:
you can always check for these, only set ENABLE_something if you want to actually use these feature.

always separate between two processes:
1. detection
2. actual enable for use